### DISCUSSION TITLE
Even Hour Intervals for Forecast Graph

### DISCUSSION THREAD

**Discussion by 13regent** [FelixdelasPozas/TrayWeather#140] [Ideas]
I think with the forecast graph, an initial  interval of 12 hours would make sense vsiually.  With intervals currently at 10.5 hours, things look a bit confusing at a first glance. See graph below: the first interval starts at 13 (00) and the second interval is 13 (21). Wouldnt it make sense for the second interval to be 14 (00)?

![Graph](https://user-images.githubusercontent.com/28539673/201505980-2fec857c-5203-4b97-81b8-b219d560db3e.jpg)

**Comment by FelixdelasPozas** [comment]
The next version (1.28.0) will fix this issue, as explained [here] (https://github.com/FelixdelasPozas/TrayWeather/issues/141).
